Dockerfile

```diff
@@ -2,4 +2,24 @@ FROM openjdk:8-jre
 
 COPY ./modules/application/target/*SNAPSHOT.jar /truthtree/
 
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y git && \
+    curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
```

Comment:
Is it node 8? can we install 10+? It's because we're building with latest node on front end travis ci, so it'll keep thing consistent. We might use some feature that not available in node 8 and that would break prod build

Reply:
ACK